__tests__/api.ts

```diff
@@ -58,11 +99,21 @@ describe('fetchApi()', () => {
   })
 
   test('transfers meta data to fetch()', () => {
+    server.use(
+      rest.get('https://api.serlo.org/something', (_req, res, ctx) => {
+        return res.once(ctx.status(200), ctx.body('<api-result>'))
+      })
+    )
     const apiRequest = fetch.getRequestTo('https://api.serlo.org/') as Request
     expect(apiRequest.headers.get('Content-Type')).toBe('application/json')
```

Comment:
This will not work (fetch has no method `getRequestTo`). You need to store the `_req` object which you get when you mock the code. Example

```
let request: RequestObject // check in mwjso which type `req` has
    server.use(
      rest.get('...', (req, res, ctx) => {
         request = req
         return res.once(ctx.status(200), ctx.body('<api-result>'))
      })
    )
```

Think about a good solution here.

Reply:
```
requestHandler = returnResponseText("<api-result>")
...
requestHandler.getRequest()
```